package.json

```diff
@@ -14,7 +14,7 @@
   },
   "peerDependencies": {
     "@bedrock/quasar": "^8.0.1",
-    "@bedrock/vue-credential-card": "^5.0.0",
+    "@bedrock/vue-credential-card": "github:digitalbazaar/bedrock-vue-credential-card#generic-credential-component",
```

Comment:
This needs to be a released version before merging.

Reply:
Agreed. This library has been renamed and moved to here https://github.com/digitalbazaar/bedrock-vue-vc/pull/1.

Reply:
resolved in https://github.com/digitalbazaar/bedrock-vue-wallet/pull/13/commits/b290b8b90a2cbad40b09691fea46fc6f4f845ec8